### DISCUSSION TITLE
Picture in Picture - Display across workspaces

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#770] [Ideas]
### What feature would you like to see?

Currently the PIP mode works great, but when switching workspaces (atleast on Windows), the PIP is no longer visible.

In Windows, there is an option in Task View which is "Show this window on all desktops".

It would be great if that is toggled by default, and there is no user input required.

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/670_

**Comment by TheLastHuntsman** [comment]
Currently on Windows, using the PiP mode, I can still see the popup as I swap between workspaces. 

However, when clicking "back to tab" on the PiP spawned in workspace 1, when currently in workspace 2, it will spawn a new tab in workspace 2 until workspace 1 has been accessed again (then the tab in workspace 2 removes itself). 

IMO the "flow" from "back to tab" should be to take the user back to the workspace the PiP spawned from initially.

**Comment by benmandr** [comment]
May not be super relevant, but for those using Sway who are interested, you can achieve it with `floating` and `sticky` options.
Here's my config if interested as a reference:
`for_window [title="Picture-in-Picture"] floating enable, sticky enable, resize set 285 160, move position 1615 861`
(I also use resize and move options to always PIP in the bottom right corner)
